### PR TITLE
870 non integer grading

### PR DESCRIPTION
**Pull request by TCMeldrum**
Closes https://github.com/UserOfficeProject/user-office-project-issue-tracker/issues/870

## Description

Add the ability to add non-intergrading to SEPs. If the precision is set to 1 then the normal drop-down menu is used else it is a normal number input. 

One thing to note is I have changed the average grade and stdev of the grade on the proposals view to 2dp as it would be strange to have it 1 dp while the grades precision was 2 dp. Wasn't sure of the best solution here

I also fixed a bug with the awaiting the db patches to run.

## Motivation and Context

Our instrument sci use decimal point :) 

## How Has This Been Tested

Manually

## Tests included/Docs Updated?

- [ ] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: apps/user-office-frontend/src/components/review/ProposalGrade.tsx [hunk 1]
```diff
@@ -50,6 +51,13 @@ const ProposalGrade = ({
   const [isSubmitting, setIsSubmitting] = useState(false);
   const [numberOfChars, setNumberOfChars] = useState(0);
   const hasAccessRights = useCheckAccess([UserRole.USER_OFFICER]);
+  const { settingsMap } = useContext(SettingsContext);
+
+  const gradeDecimalPoints = parseFloat(
+    settingsMap.get(SettingsId.GRADE_PRECISION)?.settingsValue?.valueOf() ?? '1'
+  );
+
+  console.log(gradeDecimalPoints);
```
Comment (bolmsten) [inline apps/user-office-frontend/src/components/review/ProposalGrade.tsx:60]: Remove console.log

**Comment by bolmsten** [comment]
looks good!

**Comment by bolmsten** [comment]
Can we write some E2E tests for this

**Comment by TCMeldrum** [comment]
Cool will do

**Review by simonfernandes — APPROVED** [review]

**Review by RasmiaKulan — APPROVED** [review]

**Review by martin-trajanovski — APPROVED** [review]
Looks good to me

**Pull request merged**